e2eAIOK/DeNas/nlp/bert_trainer.py

```diff
@@ -39,7 +36,12 @@ def _pre_process(self):
             custom_ops_thop = customer_ops_map_thop()
             macs_thop, _ = profile(self.model, inputs=(inputs,), custom_ops=custom_ops_thop)
             logging.info("(THOP) MACs: %.2f" % (macs_thop/(1000**3)))
+        
+        #whether to use the KD from MA
         if 'teacher_model' in self.cfg and self.cfg.teacher_model != 'None':
+            from e2eAIOK.DeNas.nlp.model_builder_denas_nlp import ModelBuilderNLPDeNas
```

Comment:
@tianyil1 , you may add a print here if aiok-modeladaptor is not installed.

Reply:
Have refined it. Thanks. @xuechendi